dask/array/core.py

```diff
@@ -2125,7 +2129,9 @@ def from_array(x, chunks, name=None, lock=False, asarray=True, fancy=True,
                     dtype=x.dtype)
         dsk[original_name] = x
 
-    return Array(dsk, name, chunks, dtype=x.dtype)
+    meta = np.empty_like(x, shape=(0,))
```

Comment:
I recommend instead using slicing here like

```python
meta = x[tuple(slice(0, 0, None) for _ in x.ndim)]
```

This is for two reasons:

1.  It would be nice to maintain the dimensionality of the array, so that `len(self.shape) == len(self._meta.shape)`
2.  We probably can't use `np.empty_like(..., shape=...)` in mainline code for a while.  We have to support older versions of NumPy in the mainline codebase.